src/components/icon-button/icon-button.stories.tsx

```diff
@@ -27,10 +27,10 @@ export const Default: Story = {
       control: "boolean",
       defaultValue: false,
     },
-    icon: {
-      options: ["chat"],
-      control: "text",
-      defaultValue: "chat",
-    },
+    // icon: {
+    //   options: ["chat"],
+    //   control: "text",
+    //   defaultValue: "chat",
+    // },
```

Comment:
It's giving a type error when I leave it in. 
I didn't modify prod code, but did change the stories to avoid issues.
I'll remove this part of the code for the story, and we can re-add what makes sense if we re-visit this stories file.

Reply:
I've modified the story to use SVG icons, to align it with the interface. It's properly being displayed in Storybook now :)

Reply:
great, thanks!